public/pages/CreateMonitor/components/CustomSteps/CustomSteps.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { Component } from 'react';
+import { EuiPanel, EuiAccordion, EuiTitle } from '@elastic/eui';
+import './CustomSteps.scss';
+
+class CustomSteps extends Component {
+  constructor(props) {
+    super(props);
+    this.stepRefs = {};
+    this.state = {
+      lineHeights: {}
+    };
+  }
+
+  componentDidMount() {
+    // Calculate line heights once at startup
+    setTimeout(() => {
+      this.updateLineHeights();
+    }, 100);
+  }
+
+  updateLineHeights = () => {
+    const { steps } = this.props;
+    const newLineHeights = {};
+    
+    for (let i = 0; i < steps.length - 1; i++) {
+      const currentRef = this.stepRefs[`step-${i}`];
+      const nextRef = this.stepRefs[`step-${i + 1}`];
+      
+      if (currentRef && nextRef) {
+        const currentCircle = currentRef.querySelector('.custom-step-circle');
+        const nextCircle = nextRef.querySelector('.custom-step-circle');
+        
+        if (currentCircle && nextCircle) {
+          const currentRect = currentCircle.getBoundingClientRect();
+          const nextRect = nextCircle.getBoundingClientRect();
+          
+          const distance = nextRect.top - currentRect.bottom;
+          newLineHeights[`line-${i}`] = Math.max(distance, 20); // Minimum 20px
+        }
+      }
+    }
+    
+    this.setState({ lineHeights: newLineHeights });
+  };
+
+  // Remove the handleAccordionToggle method since we don't want to recalculate
```

Comment:
Not blocking:
I don't see a `handleAccordionToggle` function, so this may be an outdated comment.